Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM registry.daymax.xyz/tm-go-deployer:latest
+
+ENV binary gogen-avro
+
+WORKDIR /app
+
+RUN update-ca-certificates --fresh
+
+ADD $binary /app/
+
+ENTRYPOINT ["./$binary"]
```

Comment:
I should really update `go-skel` (or you can!) but this won't work. Turns out interpolation does not work for `ENTRYPOINT` or `CMD` in dockerfiles.